PROTOCOL.md

```diff
@@ -1179,6 +1182,29 @@ When Row Tracking is enabled (when the table property `delta.enableRowTracking`
   In particular, writers should set `delta.rowTracking.preserved` in the `tags` of the `commitInfo` action to `true` if no rows are updated or copied.
   Writers should set that flag to false otherwise.
 
+# VACUUM Protocol Check
+
+The `vacuumProtocolCheck` ReaderWriter feature ensures consistent application of reader and writer protocol checks during `VACUUM` operations, addressing potential protocol discrepancies and mitigating the risk of data corruption due to skipped writer checks.
```

Comment:
Can we add `vacuumProtocolCheck` to [Valid Feature Names in Table Features](https://github.com/delta-io/delta/blob/master/PROTOCOL.md#valid-feature-names-in-table-features) in appendix?

Reply:
+1 thank you for catching this

Reply:
Added.